src/helpers/validation.ts

```diff
@@ -0,0 +1,21 @@
+import * as yup from "yup";
+
```

Comment:
```suggestion
import { OptionalObjectSchema } from "yup/lib/object";
import { AnyObject } from "yup/lib/types";

export type AnyYupSchema = OptionalObjectSchema<AnyObject>;
```

such way we remove type "any" in middlewares